recipes/grakel/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "grakel" %}
+{% set version = "0.1.8" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/grakel-{{ version }}.zip
+  sha256: 44cbfbeadc3d7cf8de22fdb59081b2fc36b9a2cbdeedd848ac6ea85208009a31
+
+build:
+  number: 0
+  skip: true   # [py>=40]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - numpy
+    - python
+    - cython
+  run:
+    - cython >=0.27.3
+    - future >=0.16.0
+    - joblib
+    - nose >=1.1.2
+    - numpy >=1.14.0
+    - python
+    - scikit-learn >=0.19
+    - six >=1.11.0
+
+test:
+  imports:
+    - grakel
+    - grakel.datasets
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://ysig.github.io/GraKeL/
+  summary: A scikit-learn compatible library for graph kernels
```

Comment:
Should be able to remove this duplicated `about` block

Reply:
I removed the secondary about block.